test/fixtures/yarn-pnp/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "yarn-pnp-test",
```

Comment:
Do we still need this test here - can we not just have a simple resolve hook unit test rather now?

Reply:
removed 👍